### PR TITLE
Disallow anonymous PUT /transfer/:id

### PR DESCRIPTION
**Pull request by lumberj**

### CODE REVIEW

**Comment by lumberj** [comment]
cc: @sentientwaffle

File: test/putTransferSpec.js [hunk 1]
```diff
@@ -394,6 +410,8 @@ describe('PUT /transfers/:id', function () {
 
     yield this.request()
       .put(transfer.id)
+      .auth('alice', 'alice')
+      .send(transfer)
```
Comment (clark800) [inline test/putTransferSpec.js]: Duplicate

Comment (lumberj) [inline test/putTransferSpec.js]: Fixed

**Comment by sentientwaffle** [comment]
LGTM

**Pull request merged**